packages/bitcoin-esplora-swap-find-provider/lib/BitcoinEsploraSwapFindProvider.ts

```diff
@@ -10,18 +10,28 @@ type PaymentVariants = {
 }
 
 export default class BitcoinEsploraSwapFindProvider extends NodeProvider {
-  constructor(url: string) {
+  _useRest: boolean
```

Comment:
it looks like you are trying to support 2 apis with the same code, technically all the requests are rest calls
would it be better to create another provider that extends this and implements the `findAddressTransaction` differently?